### PR TITLE
[CIAPP-3783] Add job logs collection mention to exploring pipelines page

### PR DESCRIPTION
**Pull request by soyguijarro**
### What does this PR do?

Add job logs collection mention to exploring pipelines page

### Motivation

 This is particularly needed to link to it from the UI (see context in https://datadoghq.atlassian.net/browse/CIAPP-3783).

### Preview

https://docs-staging.datadoghq.com/ramon/ci-app_job-log-collection/continuous_integration/explore_pipelines/#explore-connections-to-logs

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/explore_pipelines.md [hunk 1]
```diff
@@ -31,34 +31,39 @@ Get insights on the selected pipeline such as total and failed executions over t
...
 
-Click the CI provider link (`gitlab-ci gitlab.pipeline > documentation` in the following image) to investigate the Resource, Service, or Analytics page for the pipeline, stage, or job specifically. You can also find complete tags information and links to related log events and network monitoring events.
+Click the CI provider link (`gitlab-ci gitlab.pipeline > documentation` in the following image) to investigate the Resource, Service, or Analytics page for the pipeline, stage, or job specifically. You can also find complete tags information and links to network monitoring events.
 
 {{< img src="ci/ci-pipeline-execution.png" alt="Trace info for pipeline execution" style="width:100%;">}}
 
+### Explore connections to logs
+
+If job log collection is enabled for the pipeline provider, you'll also be able to find related log events in the Logs tab of the pipeline execution view. Note that job log collection is currently only supported for [Jenkins][2].
```
Comment (fermayo) [inline content/en/continuous_integration/explore_pipelines.md:44]:
```suggestion
If job log collection is supported and enabled for the CI provider, related log events can be found in the _Logs_ tab of the pipeline execution view.

**Note**: job log collection is currently only supported for [Jenkins][2].
```

**Review by fermayo — APPROVED** [review]

File: content/en/continuous_integration/explore_pipelines.md [hunk 2]
```diff
@@ -31,34 +31,41 @@ Get insights on the selected pipeline such as total and failed executions over t
 
 The pipeline execution list at the bottom shows all the times that pipeline (or its stages or jobs) ran during the selected time frame, for the selected branch. Use the facets on the left side to filter the list to exactly the pipelines, stages, or jobs you want to see.
 
-### Explore connections to services, resources, logs, and network events
+### Explore connections to services, resources and network events
```
Comment (jtappa) [inline content/en/continuous_integration/explore_pipelines.md:34]: We still want that Oxford comma ;)
```suggestion
### Explore connections to services, resources, and network events
```

File: content/en/continuous_integration/explore_pipelines.md [hunk 3]
```diff
@@ -31,34 +31,41 @@ Get insights on the selected pipeline such as total and failed executions over t
...
-Click the CI provider link (`gitlab-ci gitlab.pipeline > documentation` in the following image) to investigate the Resource, Service, or Analytics page for the pipeline, stage, or job specifically. You can also find complete tags information and links to related log events and network monitoring events.
+Click the CI provider link (`gitlab-ci gitlab.pipeline > documentation` in the following image) to investigate the Resource, Service, or Analytics page for the pipeline, stage, or job specifically. You can also find complete tags information and links to network monitoring events.
 
 {{< img src="ci/ci-pipeline-execution.png" alt="Trace info for pipeline execution" style="width:100%;">}}
 
+### Explore connections to logs
+
+If job log collection is supported and enabled for the CI provider, related log events can be found in the _Logs_ tab of the pipeline execution view.
+
+**Note**: job log collection is currently only supported for [Jenkins][2].
```
Comment (jtappa) [inline content/en/continuous_integration/explore_pipelines.md:46]:
```suggestion
**Note**: Job log collection is only supported for [Jenkins][2].
```

**Review by jtappa — CHANGES_REQUESTED** [review]
Just some small feedback, thanks!

**Comment by soyguijarro** [comment]
> Just some small feedback, thanks!

Applied. Read to merge now, thank you!

**Review by jtappa — APPROVED** [review]

**Pull request merged**